src/Components/Catalog/Main/Main.js

```diff
@@ -1,30 +1,66 @@
+import React from 'react';
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+
+
 import style from '../../../styles/Main.module.css'
 
 import Poster from '../Poster/Poster';
+import Paginations from '../Pagination/Pagination'
+import { keys } from '../../../keys';
 
-import axios from 'axios';
-import { useState, useEffect } from 'react';
-
-function Main(){
+const Main = () => {
     const [films,setFilms] = useState([])
     const [filmsPerPage] = useState(8)
     const [currentPage, setCurrentPage] = useState(1)
     const [loading,setLoading] = useState(false)
-
+    
     useEffect(() => {
+
         const getFilms = async () => {
-            setLoading(true)
-            const res = await axios.get('http://www.omdbapi.com/?i=tt3896198&apikey=fb59bcb5')
-            setFilms(res.data)
-            setLoading(false)
-        }
+            let endpoints = []
+            setLoading(true)          
 
+            for( let i = firstFilmIndex; i < lastFilmIndex; i++){
+                endpoints.push(keys[i])
+            }
```

Comment:
try to use slice in this case. It will be more readable

Reply:
done